server/filestore.go

```diff
@@ -2634,7 +2638,7 @@ func (fs *fileStore) ConsumerStore(name string, cfg *ConsumerConfig) (ConsumerSt
 		name: name,
 		odir: odir,
 		ifn:  path.Join(odir, consumerState),
-		fch:  make(chan struct{}),
+		fch:  make(chan struct{}, 1), // Make buffered on purpose.
```

Comment:
This is the one that when I search all references in VS Code, I can't find any. It seems to me that `fch` in this struct is never used?

Reply:
I think you are right. Weird staticcheck did not catch it.